lib/fireblocks/createtransaction.go

```diff
@@ -43,19 +43,17 @@ func newTransactionRequest(opt TransactionRequestOptions) createTransactionReque
 	return createTransactionRequest{
 		Operation: "RAW",
 		Note:      "testing transaction",
-		AssetID:   "ETH",
 		Source: source{
 			Type: "VAULT_ACCOUNT",
 			ID:   "0",
 		},
-		Destination: &destination{
-			Type: "VAULT_ACCOUNT",
-		},
 		CustomerRefID: "",
 		ExtraParameters: &extraParameters{
 			RawMessageData: rawMessageData{
+				Algorithm: "MPC_ECDSA_SECP256K1",
 				Messages: []UnsignedRawMessage{{
-					Content: hex.EncodeToString(contentHash[:]),
+					Content:        hex.EncodeToString(contentHash[:]),
+					DerivationPath: opt.Message.DerivationPath,
```

Comment:
fails with derivation path [44, 60, 0, 0] and both signing algorithms, passes with any valid `AssetID` such as `TTTT_ETH_TEST6`